app/Main.hs

```diff
@@ -1,19 +1,26 @@
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
```

Comment:
Documented my reasoning for this here: https://github.com/channable/vaultenv/pull/35#discussion_r167576869 (in case you  miss the collapsed comment)

Reply:
Saving a few characters is not really worth turning on a language extension imo (you probably introduced more chars than you saved with this pragma). But if you prefer it this way, then that's fine with me too.

Reply:
Yeah, it's not really about the net characters saved, but about the overhead of having to come up with names for the temporary bindings (small papercut). When reading the code, this way you also know that the binding will not be used elsewhere. The code becomes "about the logic/dataflow" instead of "about the names given tot things"

`</philosophical-rambling>`